README.md

```diff
@@ -8,7 +8,7 @@ Both win_flex and win_bison are based on upstream sources but depend on system l
 * 2.5.x versions include GNU Bison version 3.x.x
 
 ## License
-Flex uses a [BSD license](flex/src/COPYING), GNU Bison is [licensed under the GNU General Public License (GPLv3+)](bison/src/COPYING).  
+Flex uses a [BSD license](flex/src/COPYING), GNU Bison is [licensed under the GNU General Public License (GPLv3+)](bison/src/COPYING).
```

Comment:
.md files should not be whitespace-trimmed at the end. Those two trailing spaces lead to a line break, otherwise it would be a block text, please revert this part.

Reply:
Its trimmed by trimTrailingWhitespace, I'll reinsert them and add `md` files as exception